lib/cdo/shared_constants.rb

```diff
@@ -645,4 +645,9 @@ module SharedConstants
   CENSUS_CONSTANTS = OpenStruct.new(
     {CURRENT_CENSUS_SCHOOL_YEAR: 2023}
   )
+
+  # Current song manifest file name for Dance Party. Note that different manifests
+  # can be tested using query params (?manifest=...), but once this value is updated
+  # the default manifest will change for all users.
+  DANCE_SONG_MANIFEST = 'songManifest2023_v4.json'
```

Comment:
What do you think of `DANCE_SONG_MANIFEST_FILENAME`?

Reply:
Yeah that's more descriptive, will update!